netbox/overlays/ocp-staging/externalsecrets/oauth2-secret.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: "kubernetes-client.io/v1"
+kind: ExternalSecret
+metadata:
+  name: oauth2-secret
+spec:
+  backendType: secretsManager
+  data:
+    - key: cluster/ocp-prod/netbox/sso-clientsecret-moc
+      name: OAUTH2_PROXY_CLIENT_SECRET
+    - key: cluster/ocp-prod/netbox/oauth2-cookie-secret
+      name: OAUTH2_PROXY_COOKIE_SECRET
```

Comment:
I guess we are using the same sso-client as ocp-prod. I am okay with it, just wanted to point it out

Reply:
Yeah, I figured it made sense to use the same auth in this case.